### PR TITLE
refactor event

### PR DESCRIPTION
**Pull request by haitham1337**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -79,7 +80,7 @@ Citizen.CreateThread(function()
 
                                     if not copsCalled then
                                         local pos = GetEntityCoords(PlayerPedId())
-                                        local s1, s2 = Citizen.InvokeNative(0x2EB41072B4C1E4C0, pos.x, pos.y, pos.z, Citizen.PointerValueInt(), Citizen.PointerValueInt())
```
Comment (GhzGarage) [inline client/main.lua:82]: Dude....

File: client/main.lua [hunk 2]
```diff
@@ -146,7 +142,7 @@ AddEventHandler('lockpicks:UseLockpick', function(isAdvanced)
                         TriggerServerEvent("evidence:server:CreateFingerDrop", pos)
                     end
                     if not copsCalled then
-                        local s1, s2 = Citizen.InvokeNative(0x2EB41072B4C1E4C0, pos.x, pos.y, pos.z, Citizen.PointerValueInt(), Citizen.PointerValueInt())
```
Comment (GhzGarage) [inline client/main.lua:149]: Dude............

File: client/main.lua [hunk 3]
```diff
@@ -164,7 +160,7 @@ AddEventHandler('lockpicks:UseLockpick', function(isAdvanced)
...
```
Comment (GhzGarage) [inline client/main.lua:167]: Dude.......................

**Comment by GhzGarage** [comment]
You don't need the pointer value stuff

**Comment by haitham1337** [comment]
@GhzGarage true ?

**Pull request merged**